### ISSUE TITLE
chaos ch1 : quests marker not showed

### ISSUE THREAD

**Issue by zak68** [WarEmu/WarBugs#7166]
"the eidolon"
"virixia darkwhisper"
"magdr the soulreaper"
are not showing the greeen marker but will grant quest if clicked.

**Comment by Luth-RoR** [comment]
Needs confirmation from DB dev if it can be fixed atm.

**Comment by Soulwhisper** [comment]
You mean green icon over head?

**Comment by Luth-RoR** [comment]
Yes, i think so. Last info about that is iirc that it can't be fixed atm.
I even remember it sometimes didn't work on the mythic server too, so i'm not sure if maybe client issue.

**Comment by zak68** [comment]
opened this ticket since i've always have seen that icon on those npc.

**Comment by Luth-RoR** [comment]
Ok, thanks for clarification.

**Comment by zak68** [comment]
tried creating another pg and upon reaching lvl 3 those nps don't show the quest icon ; relogging will fix the behaviour.

**Comment by Luth-RoR** [comment]
Maybe related to the new quest chain concept and/or altered level req. for quests.

**Comment by Ryanfawkez** [comment]
Not related to quest chain concept. Destruction start zone was unchanged; only Order was. Quest icons on UI seems to be related to the level 1-4 inability to see Tome of Knowledge entries.

**Comment by Luth-RoR** [comment]
I can barely remember that i had this issue very rarely on the mythic server too.
Maybe some problem related to the 1.4.8 client.
